test/click-handler.test.js

```diff
@@ -10,6 +10,7 @@ describe('click-handler', () => {
   let resolveFake;
 
   sandbox.stub(window.chrome.runtime, 'sendMessage');
+  window.console.error = () => {}; // Suppress fake onClick errors
```

Comment:
Before, the `return console.error(err);` line in `onClick` wasn't getting run because `sendMessage` was stubbed, so `tryLoad` never called its callback. Now, `console.error` gets called because fetch is an async function, so we just need to silence it here (to avoid having the tests print useless output). See my other comment for details about the stubbing: https://github.com/OctoLinker/browser-extension/pull/306/files#r104506996).